### PR TITLE
Backed-up gsdocs_2019 and created gsdocs_2020 page, with an example

### PR DESCRIPTION
**Pull request by agheata**
This PR does what the title explains, without a link yet to the 2019 pages because the ROOT project we had last year was anyway not linked there. I added CERN as example organization, ROOT as example project, plus a generic proposal of a documentation project idea. This contains explanations on what could be the content. After merging this PR, we plan advertising Season of Docs 2020 to the community, sharing the info that we have and giving a guideline.

### CODE REVIEW

File: _gsdocs-proposals/2020/proposal_ROOTgeneric.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Generic Proposal Title
+layout: gsoc_proposal
+project: ROOT
```
Comment (xvallspl) [inline _gsdocs-proposals/2020/proposal_ROOTgeneric.md:4]: ROOT already confirmed?

File: gsdocs/2019/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
```
Comment (xvallspl) [inline gsdocs/2019/index.md:1]: This file is under 2019, don't know if that's ok

File: gsdocs/guideline.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+title: "HSF GSoC Mentor Guidelines"
+author: "Benedikt Hegner"
+layout: default
+---
+
+# How to Add a New project in HSF GSDocs
+
+## Instructions for Adding a New Proposal
+
+ * Option A: email GSoC [administrators](mailto:hsf-gsdocs-admin@googlegroups.com) (currently: Andrei Gheata, Antoine Pérus and Xavier Valls)
```
Comment (xvallspl) [inline gsdocs/guideline.md:11]: Should this still be an option? Shouldn't we do everything over github?

File: _gsdocs-proposals/2020/proposal_ROOTgeneric.md (hunk 1 above)
Comment (agheata) [reply _gsdocs-proposals/2020/proposal_ROOTgeneric.md:4]: No, this is just as a template, clearly stated now in the "For HSF projects and mentors" section. There is also a guideline for sending proposals now there

File: gsdocs/2019/index.md (hunk 2 above)
Comment (agheata) [reply gsdocs/2019/index.md:1]: This is for the last year page

Comment (agheata) [reply gsdocs/2019/index.md:1]: Which did not have links to the projects anyway

File: gsdocs/guideline.md (hunk 3 above)
Comment (agheata) [reply gsdocs/guideline.md:11]: Not sure, time is short, we need to consider also things sent by mail

**Pull request merged**